src/subdomains/core/payment-link/controllers/payment-link.controller.ts

```diff
@@ -125,6 +130,14 @@ export class PaymentLinkController {
 
   // --- PAYMENT --- //
 
+  @Get('recipient')
+  @ApiExcludeEndpoint()
+  @ApiQuery({ name: 'id', description: 'Route ID or label', required: true })
+  async getPaymentRecipient(@Query('id') id: string): Promise<SellDto> {
+    const sellRoute = await this.sellService.getPaymentRoute(id);
+    return this.sellController.toDto(sellRoute);
+  }
```

Comment:
This sell DTO is a legacy thing that nobody uses 🤣  I would create a new one (e.g. `PaymentRecipient`) with the required information.